sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs

```diff
@@ -8499,6 +8946,85 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.InstancePoolVcoresCapability>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct InstanceRole : System.IEquatable<Azure.ResourceManager.Sql.Models.InstanceRole>
```

Comment:
Too generic

Reply:
Renamed to DistributedAvailabilityGroupManagedInstanceRole